daemon/images/images.go

```diff
@@ -187,12 +184,10 @@ func (i *ImageService) Images(imageFilters filters.Args, all bool, withExtraAttr
 		}
 
 		if withExtraAttrs {
-			// lazily init variables
-			if summaryMap == nil {
+			// Lazily init variables
+			if summaryMap == nil && allContainers == nil {
```

Comment:
Hmm.. .my IDE doesn't like that it's now checking for the _combination_ of both being `nil`. "in theory" it's right (as in; `allContainers` may not be nil but `summaryMap` may be. That said, the warning itself is legit; it _is_ important to check that `summaryMap` is not nil here, otherwise we'd panic, so perhaps reverting to just checking if `summaryMap` is nil is ok here.

<img width="769" alt="Screenshot 2021-06-29 at 09 58 07" src="https://user-images.githubusercontent.com/1804568/123760249-f58e5b80-d8c0-11eb-83f0-40fc71ee6809.png">